### ISSUE TITLE
1.4 template changes broke semver

### ISSUE THREAD

**Issue by bebraw** [ampedandwired/html-webpack-plugin#51]
Just a quick, friendly fyi. I helped @ericwooley to debug `html-webpack-plugin` to work with https://github.com/mvilrokx/react-starter-template . The problem was that it depended on `html-webpack-plugin` using `^1.1.0` (major versions). As per semver that will match to major versions (ie. 0.1, 0.2, ... 1.0, 2.0, ...). You can try http://semver.npmjs.com/ to see how it behaves.

The problem is that somewhere between 1.1 and the current version (likely 1.4) backwards compatibility was broken. Due to the way matching works that caused breakage here.

So in short bump major version for backwards incompatible changes. That will keep dependants happy. :+1: 

I'm not sure what would be a good fix but maybe there's something that could be done. Hope this helps.

**Comment by jantimon** [comment]
Actually 1.4. should be backwards compatible. 
The error was a missing `new Error()` and was fixed by #45.
We should release a 1.4.1 version and roll this fix out

**Comment by bebraw** [comment]
Ah, ok. Great to hear!

**Comment by jantimon** [comment]
Will be fixed as soon as @ampedandwired released 1.5.

**Comment by bebraw** [comment]
1.5 seems to be out so I suppose it's safe to close this.
